### DISCUSSION TITLE
Opportunity to set Volume Control entity in Media Player card

### DISCUSSION THREAD

**Discussion by schmierlappe** [Clooos/Bubble-Card#573] [Feature requests]
Would be nice if it could be possible to add another entity for volume slider control in media player card.

Example:

I have a Apple TV connected with a Sonos Box. So main entity in card config is Apple TV media player but for volume control I want to have sonos entity in one card.

**Comment by Clooos** [comment]
Hi, I will not add this feature because you can do that by creating an universal media player in your YAML config, here's how:

https://www.home-assistant.io/integrations/universal/
